microservice/challenge-2/HELP.md

```diff
@@ -0,0 +1,140 @@
+# B-Yond, The second challenge, Save the Humans
+
+Bots started an invasion to our planet. Regarding the latest research they leave some patterns in their reviews on amazon. Here we provde a REST API on the top of a Machine Learning microservice. It supposed to finds this patterns...
+We've downloaded and mixed some datasets from here: http://jmcauley.ucsd.edu/data/amazon/
+That data is being used to prepare the dictionary.txt file. dictionary.txt is the experience we make for the machine about human reviews.
+
+## Architecture
+
+We've used Microservices architecture. The services is stateless. So you can easily bring more instances up to scale out.
+This service supposed to provide a low-level API. So it needs to be deployed behind firewalls, HA and non-blocking high-level APIs.
+
+## Getting Started
+
+1. Compile and buid the jar file:
+```
+mvn clean install -DskipTests
+```
+
+You can run service by one of the following ways:
+```
+mvn spring-boot::run
+```
+
+OR
+
+Starting the project by building a docker image and running it withing the container:
+```
+docker-compose up
+```
+
+Then access the API online document using the following address:
+```
+http://localhost:8080/swagger-ui.html
+```
+
+Note:
+After running service it starts to train itself. It takes a fe
+
+### Prerequisites
+
+The learner needs at least 2GB ram. It uses a dictionary that downloaded from here:
```

Comment:
you missed that URL ?

Reply:
The URL:
 http://jmcauley.ucsd.edu/data/amazon/